modules/apps/commerce/commerce-address-web/src/main/resources/META-INF/resources/country/regions.jsp

```diff
@@ -84,18 +84,18 @@ CommerceRegionsDisplayContext commerceRegionsDisplayContext = (CommerceRegionsDi
 				searchContainer="<%= commerceRegionsDisplayContext.getSearchContainer() %>"
 			>
 				<liferay-ui:search-container-row
-					className="com.liferay.commerce.model.CommerceRegion"
-					keyProperty="commerceRegionId"
-					modelVar="commerceRegion"
+					className="com.liferay.portal.kernel.model.Region"
+					keyProperty="regionId"
+					modelVar="region"
 				>
 
 					<%
 					PortletURL rowURL = renderResponse.createRenderURL();
 
 					rowURL.setParameter("mvcRenderCommandName", "/commerce_country/edit_commerce_region");
 					rowURL.setParameter("redirect", currentURL);
-					rowURL.setParameter("commerceCountryId", String.valueOf(commerceRegion.getCommerceCountryId()));
-					rowURL.setParameter("commerceRegionId", String.valueOf(commerceRegion.getCommerceRegionId()));
+					rowURL.setParameter("commerceCountryId", String.valueOf(region.getCountryId()));
+					rowURL.setParameter("commerceRegionId", String.valueOf(region.getRegionId()));
```

Comment:
What do you think about renaming these parameters? It might be a simple find and replace.